content/src/cache.js

```diff
@@ -0,0 +1,77 @@
+///////////////////////////////////////////////////////////////////////////
+// CACHE SUPPORT
+///////////////////////////////////////////////////////////////////////////
+
+var $              = require('jquery'),
+    filetype       = require('filetype'),
```

Comment:
Don't include any of these dependencies.  Only include the ones that this module actually needs.